integration/spark2/src/main/scala/org/apache/spark/sql/events/MergeIndexEventListener.scala

```diff
@@ -67,17 +64,9 @@ class MergeIndexEventListener extends OperationEventListener with Logging {
               carbonTable.getTablePath,
               carbonTable, false)
             // clear Block dataMap Cache
-            clearBlockDataMapCache(carbonTable, Seq(loadModel.getSegmentId))
+            MergeIndexUtil.clearBlockDataMapCache(carbonTable, Seq(loadModel.getSegmentId))
```

Comment:
not need to clear the cache which is not exists

Reply:
this clear cache is only for the segments we merged.  Not on the whole table. This clear will clean the unused cache objects.

Reply:
for a loading , before commit, it  cached nothing of this segment

Reply:
It is better to have this to avoid unknown impacts. In case of child table creation/load, it will query on the main table load. This is not so costly as the clear is only on the segments not on the whole table.